### PR TITLE
Updated Name of Countries and removed duplicate with original official names.

### PR DESCRIPTION
**Pull request by Ovalelephant35**
Updates Name of countries to new official names.
This PR closes https://github.com/sugarlabs/countries-activity/issues/22
Sources :-
Official Names of the United Nations Membership
https://www.un.int
https://worldpopulationreview.com

### CODE REVIEW

**Comment by SohamKukreti** [comment]
Thank you for the pull request. 
There are a few issues with your pull request:
1. Please comply with imperative tense recommendation and others in how to make commits.
2. Along with the names, the order of flags need to be updated too.
3. The latlon.txt file would need to be updated too.
4. Try to maintain the lexicographical order of countries (North Macedonia needs to placed correctly).

**Comment by Ovalelephant35** [comment]
Maintained order across all files.

**Comment by SohamKukreti** [comment]
Thank you for updating the flags, however there might be some mistake in the order as wrong flag image is displayed for each country:

image

and also, when I click on a particular flag in the flags of the world page, It takes me to the wrong country:

image
image

**Comment by Ovalelephant35** [comment]
It was highly possible for this to happend if you have time see where it is wrong I will also see for it.

File: data/countries.txt [hunk 1]
```diff
@@ -120,7 +116,8 @@ Mauritania
 Mauritius
 Mayotte
 Mexico
-Micronesia,Federated States of Micronesia
+Micronesia
+Federated States of Micronesia
```
Comment (chimosky) [inline data/countries.txt:121]: Why two entries?

File: data/latlon.txt
Comment (chimosky) [inline data/latlon.txt:1]: Why is this being updated? Did the countries move?

File: data/countries.txt (hunk 1 above)
Comment (Ovalelephant35) [reply data/countries.txt:121]: Yes they are now to seperate states according to UN data everything is based on doc links i provided up.

File: data/latlon.txt
Comment (Ovalelephant35) [reply data/latlon.txt:1]: Yes many of the countries have changed there name in 20 years.

File: data/countries.txt (hunk 1 above)
Comment (Ovalelephant35) [reply data/countries.txt:121]: then there might be ambiguity regarding keeping that in same line though there flags are same.
same goes with turkey/turkiye

File: data/latlon.txt
Comment (chimosky) [reply data/latlon.txt:1]: Yes the PR says names changed but coordinates are being changed too.

Comment (Ovalelephant35) [reply data/latlon.txt:1]: Yes, changed the flags also as it was mentioned above with name change flag change also has to be done and it led to mismatch.

Comment (Ovalelephant35) [reply data/latlon.txt:1]: https://github.com/sugarlabs/countries-activity/pull/25
Kindly review this pr done by @SohamKukreti 
Thankyou.

Comment (SohamKukreti) [reply data/latlon.txt:1]: Yeah, in the original PR, the older country names had been removed from countries.txt and newer ones were added in such a manner that the order of the countries was disturbed. Each country is tied to a flag and its numbering depends upon the order in which it is present in countries.txt. Thus I told @Ovalelephant35 to update the flags according to the new order they had set. The latlon file also depends upon the ordering of the country names, thus it had to be updated too.

Comment (chimosky) [reply data/latlon.txt:1]: Okay then.

**Pull request closed without merging**